include/aspect/melt.h

```diff
@@ -107,6 +107,20 @@ namespace aspect
         {};
     };
 
+    /**
+     * Base class for material models to be used with melt transport enabled.
+     */
+    template <int dim>
+    class MeltInterface: public MaterialModel::Interface<dim>
+    {
+      public:
+        /**
+          * Reference permeability divided by fluid viscosity, also known as Darcy coeffcient, Units: m^2/Pa/s
+          */
```

Comment:
This is slightly imprecise (any has a typo). What about:
Reference value for the Darcy coefficient, which is defined as permeability divided by fluid viscosity.